app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -78,6 +78,9 @@ class AuthenticationFragment : Fragment() {
                                     error: WebResourceError?
                                 ) {
                                     super.onReceivedError(view, request, error)
+                                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
This [entire function](https://developer.android.com/reference/android/webkit/WebViewClient?hl=en#onReceivedError(android.webkit.WebView,%20android.webkit.WebResourceRequest,%20android.webkit.WebResourceError)) was added on API 23, no need to include the if statement here

Reply:
android studio was complaining, I'll add a required flag on the function to make it happy :)

Reply:
Android Studio does complain if you remove the version check without adding `@RequiresApi(Build.VERSION_CODES.M)` to the function, but it will build the app 🤔

Reply:
Yea shouldn't be highlighted in red if we can still build IMO